model_aug.py

```diff
@@ -103,19 +103,14 @@ def augment_batches(self, config):
         out = self.sess.run([self.aug_image_genOP], feed_dict={self.G_inputs: G_batch_images})
         #
         ## generator output images and sampled augmentation parameters
-        if save_aug_params_flag:
-          G_output_images = np.squeeze(out[0][0])
-          ChromAbParams = np.array(out[0][1])
-          BlurParams =   np.array(out[0][2])
-          ExpParams =   np.array(out[0][3])
-          NoiseParams = np.array(out[0][4])
-          ColorParams = np.array(out[0][5])
-          ## save images
-          self.save_augmented_final_images(G_output_images, G_batch_files, ChromAbParams, BlurParams, ExpParams, NoiseParams, ColorParams, epoch)
-        else:
-          ## save images
-          G_output_images = np.squeeze(out[0])
-          self.save_augmented_final_images(G_output_images, G_batch_files, [], [], [], [], [], epoch)
+        G_output_images = np.squeeze(out[0][0])
+        ChromAbParams = np.array(out[0][1])
+        BlurParams =   np.array(out[0][2])
+        ExpParams =   np.array(out[0][3])
+        NoiseParams = np.array(out[0][4])
+        ColorParams = np.array(out[0][5])
+        ## save images
+        self.save_augmented_final_images(G_output_images, G_batch_files, ChromAbParams, BlurParams, ExpParams, NoiseParams, ColorParams, n)
```

Comment:
This if/else isn't necessary, the check for save_params is already done is save_augmented_final_images